Jenkinsfile

```diff
@@ -23,7 +23,6 @@ pipeline {
         // --------------------------------------------
         APP_NAME="runtimes-inventory"  // name of app-sre "application" folder this component lives in
         COMPONENT_NAME="runtimes-inventory"  // name of app-sre "resourceTemplate" in deploy.yaml for this component
-        IMAGE="quay.io/cloudservices/runtimes-inventory"  // image location on quay
 
         IQE_PLUGINS="runtimes-inventory"  // name of the IQE plugin for this app.
         IQE_MARKER_EXPRESSION="smoke"  // This is the value passed to pytest -m
```

Comment:
for now we need to use `not full_stack` as the marker expression. Once we have more coverage in the plugin it will make sense to update it and introduce a smoke marker. But for now we pretty much want to run anything except full_stack (tests that require the full console.redhat.com env)